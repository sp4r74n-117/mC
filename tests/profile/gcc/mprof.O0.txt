function main
 called from ??
 took avg: 9180592 stddev: 0 cycles
function sum
 called from main
 took avg: 91744 stddev: 12433 cycles
