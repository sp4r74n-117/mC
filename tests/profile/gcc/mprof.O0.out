0x80489e0 0x8048a75 89648
0x80489e0 0x8048a75 89516
0x80489e0 0x8048a75 90092
0x80489e0 0x8048a75 89064
0x80489e0 0x8048a75 89820
0x80489e0 0x8048a75 87968
0x80489e0 0x8048a75 87968
0x80489e0 0x8048a75 88464
0x80489e0 0x8048a75 90044
0x80489e0 0x8048a75 87964
0x80489e0 0x8048a75 90032
0x80489e0 0x8048a75 90036
0x80489e0 0x8048a75 90056
0x80489e0 0x8048a75 90052
0x80489e0 0x8048a75 113664
0x80489e0 0x8048a75 87976
0x80489e0 0x8048a75 89460
0x80489e0 0x8048a75 89472
0x80489e0 0x8048a75 90012
0x80489e0 0x8048a75 89832
0x80489e0 0x8048a75 88544
0x80489e0 0x8048a75 90056
0x80489e0 0x8048a75 89072
0x80489e0 0x8048a75 89996
0x80489e0 0x8048a75 89524
0x80489e0 0x8048a75 89952
0x80489e0 0x8048a75 90016
0x80489e0 0x8048a75 88332
0x80489e0 0x8048a75 89900
0x80489e0 0x8048a75 87968
0x80489e0 0x8048a75 90012
0x80489e0 0x8048a75 89560
0x80489e0 0x8048a75 88504
0x80489e0 0x8048a75 88196
0x80489e0 0x8048a75 205240
0x80489e0 0x8048a75 90056
0x80489e0 0x8048a75 90040
0x80489e0 0x8048a75 90040
0x80489e0 0x8048a75 89444
0x80489e0 0x8048a75 90008
0x80489e0 0x8048a75 90052
0x80489e0 0x8048a75 89508
0x80489e0 0x8048a75 90044
0x80489e0 0x8048a75 89736
0x80489e0 0x8048a75 89500
0x80489e0 0x8048a75 87960
0x80489e0 0x8048a75 89852
0x80489e0 0x8048a75 90056
0x80489e0 0x8048a75 90072
0x80489e0 0x8048a75 89500
0x80489e0 0x8048a75 89568
0x80489e0 0x8048a75 88312
0x80489e0 0x8048a75 110108
0x80489e0 0x8048a75 89576
0x80489e0 0x8048a75 89536
0x80489e0 0x8048a75 89076
0x80489e0 0x8048a75 89512
0x80489e0 0x8048a75 88824
0x80489e0 0x8048a75 90060
0x80489e0 0x8048a75 89360
0x80489e0 0x8048a75 89852
0x80489e0 0x8048a75 89944
0x80489e0 0x8048a75 90008
0x80489e0 0x8048a75 90064
0x80489e0 0x8048a75 90020
0x80489e0 0x8048a75 89520
0x80489e0 0x8048a75 88448
0x80489e0 0x8048a75 90048
0x80489e0 0x8048a75 90024
0x80489e0 0x8048a75 119196
0x80489e0 0x8048a75 89908
0x80489e0 0x8048a75 108220
0x80489e0 0x8048a75 89604
0x80489e0 0x8048a75 89220
0x80489e0 0x8048a75 89808
0x80489e0 0x8048a75 89880
0x80489e0 0x8048a75 88356
0x80489e0 0x8048a75 88804
0x80489e0 0x8048a75 90048
0x80489e0 0x8048a75 89520
0x80489e0 0x8048a75 90020
0x80489e0 0x8048a75 88732
0x80489e0 0x8048a75 89980
0x80489e0 0x8048a75 88468
0x80489e0 0x8048a75 89980
0x80489e0 0x8048a75 89544
0x80489e0 0x8048a75 90020
0x80489e0 0x8048a75 88508
0x80489e0 0x8048a75 89980
0x80489e0 0x8048a75 89964
0x80489e0 0x8048a75 90020
0x80489e0 0x8048a75 106376
0x80489e0 0x8048a75 89480
0x80489e0 0x8048a75 89844
0x80489e0 0x8048a75 88608
0x80489e0 0x8048a75 89536
0x80489e0 0x8048a75 89428
0x80489e0 0x8048a75 89532
0x80489e0 0x8048a75 90016
0x80489e0 0x8048a75 90028
0x8048a3a 0xf75a972e 9180592
